examples/docs/package.json

```diff
@@ -6,8 +6,12 @@
     "start": "astro dev",
     "build": "astro build"
   },
+  "dependencies": {
+    "@docsearch/react": "^1.0.0-alpha.27"
+  },
   "devDependencies": {
-    "astro": "^0.19.0-next.2"
+    "@snowpack/plugin-dotenv": "^2.1.0",
+    "astro": "^0.19.0-next.fbr.1"
```

Comment:
make sure you match the version in the repo:  `"^0.19.0-next.2"` Otherwise, yarn will install the wrong version and your project wont work.